app/core/treema-ext.coco.coffee

```diff
@@ -246,6 +246,50 @@ class ImageFileTreema extends TreemaNode.nodeMap.string
     @flushChanges()
     @refreshDisplay()
 
+class GeneralFileTreema extends TreemaNode.nodeMap.string
```

Comment:
Can we make ImageFileTreema a subclass of this, or otherwise reuse code? Almost everything is the same between these two.